docker/candlepin-base/cp-test

```diff
@@ -256,16 +257,23 @@ if [ "$UNITTEST" -gt 0 ]; then
 fi
 
 if [ "$QPID" == "1" ]; then
-    echo "Setting up Qpid"
-    pushd . 
-    cd ../server/bin/qpid
-    # Do a cleanup 
-    ./configure-qpid.sh -c
-    # Reconfigure
-    ./configure-qpid.sh
-    popd 
-fi
+    if [ -d "$PROJECT_DIR/bin/qpid" ]; then
+        echo "Setting up Qpid"
+
+        pushd .
+        cd "$PROJECT_DIR/bin/qpid"
```

Comment:
This can actually be shortened to

```
pushd "$PROJECT_DIR/bin/qpid"
```

The shell will keep the current directory on the stack and you'll return to it when you run `popd`